zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -194,7 +194,12 @@ public boolean accept(File pathname) {
 
     try {
       out.reset();
-      npmCommand("install --loglevel=error");
+      String npmCommand =
+              "install --fetch-retries=2 --fetch-retry-factor=1 " +
```

Comment:
and also, That L380-L382 codes are not wrapped with `try catch TaskRunnerException`, like below code.
https://github.com/NohSeho/zeppelin/blob/b7296ca0d60ba99d6c874831330a823ccea2f2b0/zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java#L195-L209

Would better this comment be managed in other issue?